src/spid_sp_test/dump_pem.py

```diff
@@ -29,13 +30,15 @@ def _dump_pem(xml_elem, data_dir, context, use):
     [pem.append(b64[i : i + n]) for i in range(0, len(b64), n)]
     pem.append("-----END CERTIFICATE-----")
 
-    x509 = OpenSSL.crypto.load_certificate(
-        OpenSSL.crypto.FILETYPE_ASN1, base64.b64decode(b64)
-    )
+    #x509 = OpenSSL.crypto.load_certificate(
```

Comment:
@damikael please remove the comments and change the version to 1.2.14 here:
https://github.com/italia/spid-sp-test/blob/main/src/spid_sp_test/__init__.py

then we'll ahve a new release, thx!

Reply:
done